### PR TITLE
feat: Create a task

### PR DESCRIPTION
**Pull request by jenova-0**
Change log 🔮: 

- Add create task API

### CODE REVIEW

File: src/database/tasks.js [hunk 1]
```diff
@@ -45,6 +45,74 @@ function Database(server, options, next) {
...
+				const result = await connection.execute(
+					query.slice(0, -1),
+					injectable
+				);
+
+				await connection.commit();
+
+				connection.release();
+				return result;
+			} else {
```
Comment (vader-sama) [inline src/database/tasks.js:105]: No rollback is needed
Change to this
```js
else {
        connection.release();
        return null;
}
```

File: src/database/tasks.js [hunk 2]
```diff
@@ -45,6 +45,74 @@ function Database(server, options, next) {
...
+					"INNER JOIN tags ON tags.team = users.team AND tags.id = ? " +
+					"INNER JOIN statuses ON statuses.team = users.team AND statuses.order_number = 1 " +
+					"INNER JOIN teams ON teams.id = users.team AND teams.id = ? " +
+					"INNER JOIN users AS assignees ON assignees.team = users.team AND " +
+					`assignees.id IN (${assignees.map(() => "?").join(",")}) ` +
+					"WHERE users.id = ?",
+				[tag, team, ...assignees, user]
+			);
+
+			if (validation.length >= assignees.length) {
```
Comment (vader-sama) [inline src/database/tasks.js:75]: Start transaction in this `if` block
There is no need to start a transaction for reading data

**Review by vader-sama — CHANGES_REQUESTED** [review]

File: src/database/tasks.js (hunk 1 above)
Comment (jenova-0) [reply src/database/tasks.js:106]: resolve

**Review by vader-sama — APPROVED** [review]
LGTM! 🌟

**Pull request merged**